examples/info.js

```diff
@@ -29,3 +29,7 @@ console.log(err3.message);
 console.log(err3.name);
 console.log(VError.info(err3));
 console.log(err3.stack);
+
```

Comment:
I don't think this really belongs in the info example -- maybe a new examples/fullstack.js?